WordPress/Classes/ViewRelated/Blog/Blog Details/PostsCardViewController.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+/// Render a small list of posts for a given blog and post status (drafts or scheduled)
+///
+/// This class handles showing posts from the database, syncing and interacting with them
+///
+@objc class PostsCardViewController: UIViewController {
+    @objc var blog: Blog?
+
+    private let postsTableView = IntrinsicTableView()
+
+    private var viewModel: PostsCardViewModel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureView()
+        viewModel = PostsCardViewModel(tableView: postsTableView, blog: blog!)
```

Comment:
What do you think of passing a non-optional blog in the init method so we don't have to implicitly unwrap the blog object?

Reply:
Done in 8393babaf7e17796f9bca2242715377d80aef55e